### ISSUE TITLE
[Documentation] Clarify that most of these examples cannot be deployed

### ISSUE THREAD

**Issue by maxtheman** [cloudflare/python-workers-examples#11]
Hello Team CF,

I am enjoying Python workers so far. Unfortunately I misread the disclaimer on this page and wrote an app in FastAPI, which I'll now have to refactor to remove if I'm to use.

The offending line/link: "Python Workers are in open beta.
You can currently only use [built-in packages](https://developers.cloudflare.com/workers/languages/python/packages/#supported-packages) in local development. Support for deploying packages with a requirements.txt file is coming soon."

The way this is phrased reads, to me, that you can't use requirements.txt — except for the packages shown when you click on the link.

I now understand that the intended meaning is "For now, you can only use the Python Standard Library, all of the packages listed below will be supported soon."

I think the issue is with the term "built-in packages". That doesn't really have any kind of special meaning in python, I'd rephrase this page to say "Standard Library", which is what the supported packages are actually called.

The fact that this worked locally + the lack of docs clarity wasted about one work-day of my time. It's my bad for not trying to deploy earlier, but to help others, I think it would be useful to update this phrasing.

Thank you!

**Comment by mikenomitch** [comment]
Hey @maxtheman, this is good feedback. Sorry that the lack of clarity resulted in burnt time on your end.

I will PR some changes to the docs and this repo today, and see if its possible to emit a warning on `wrangler dev`.

**Comment by maxtheman** [comment]
Thanks @mikenomitch! That should help future devs.

**Comment by bitnom** [comment]
> The fact that this worked locally + the lack of docs clarity wasted about one work-day of my time. It's my bad for not trying to deploy earlier, but to help others, I think it would be useful to update this phrasing.

Thankfully this. The same thing happened to me. It was an ordeal. I was actually pretty confused by it until seeing your post on discord just now.
